h/static/scripts/directive/annotation.js

```diff
@@ -72,50 +89,98 @@ function extractDocumentMetadata(model) {
   return document_;
 }
 
-/** Copy properties from viewModel into domainModel.
-*
-* All top-level properties in viewModel will be copied into domainModel,
-* overwriting any existing properties with the same keys in domainModel.
-*
-* Additionally, the `tags` property of viewModel - an array of objects
-* each with a `text` string - will become a simple array of strings in
-* domainModel.
-*
-* @param {object} domainModel The object to copy properties to
-* @param {object} viewModel The object to copy properties from
-* @returns undefined
-*
-*/
-function updateDomainModel(domainModel, viewModel) {
-  angular.extend(domainModel, viewModel);
-  domainModel.tags = viewModel.tags.map(function(tag) {
-    return tag.text;
-  });
+/** Return `true` if the given annotation is a reply, `false` otherwise. */
+function isReply(domainModel) {
+  if ((domainModel.references || []).length > 0) {
```

Comment:
The `if` statement is redundant here, you can just write this as `return <boolean expression>`, ie. `return domainModel.references && domainModel.references.length > 0`